apis/placement/v1beta1/stageupdate_types.go

```diff
@@ -167,6 +180,13 @@ type UpdateRunSpec struct {
 	// and recorded in the status field.
 	// +kubebuilder:validation:Required
 	StagedUpdateStrategyName string `json:"stagedRolloutStrategyName"`
+
+	// ExecutionState indicates the desired execution state of the update run.
+	// When nil or "Stopped", the update run will initialize but not execute.
+	// When "Started", the update run will begin or continue execution.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=Started;Stopped
```

Comment:
why make this optional instead of having a default?

Reply:
Changed it default, but now default is Stop which seems awkward. Also thought about having the states as Active/Inactive